src/pke/include/cryptocontext.h

```diff
@@ -124,11 +124,24 @@ class CryptoContextImpl : public Serializable {
             else {
                 elemParamsPtr = cryptoParams->GetElementParams();
             }
-
+            // Check if plaintext has got enough slots for data (value)
```

Comment:
Why do we need to add the CKKS plaintext check here if it's already in MakePlaintext?

Reply:
Unfortunately we do not call MakePlaintext() here. I have a TODO comment added to change this